packages/magellan-mocha-plugin/lib/test_run.js

```diff
@@ -41,7 +41,11 @@ const MochaTestRun = function ( options ) {
 
 // return the command line path to the test framework binary
 MochaTestRun.prototype.getCommand = function () {
-	return './node_modules/.bin/mocha';
+	const mochaPath = path.dirname( require.resolve( 'mocha' ) );
```

Comment:
```suggestion
	const mochaPath = path.dirname( require.resolve( 'mocha/package.json' ) );
```
Their `main` file can change so it's better to resolve the manifest which is always under the root dir

Reply:
I'm not trying to get their `main`, but their `bin.mocha`. I guess they can change the name of the "binary" file, but that seems unlikely.

Reply:
I know but `require.resolve('mocha')` will resolve to their `main` field (or `exports`), in this case its undefined so it will resolve to `mocha/index.js` but the `main` field could be set to `lib/index.js` which would cause the rest of your code to fail. Resolving `mocha/package.json` prevents that issue since it's always at the same location

Reply:
That makes a lot of sense, thanks for explaining